package.json

```diff
@@ -34,7 +35,8 @@
     "is-valid-domain": "^0.1.6",
     "json-schema-to-ts": "^2.8.0",
     "mongoose": "^7.0.4",
-    "pino": "^8.11.0"
+    "pino": "^8.11.0",
+    "pkijs": "^3.0.14"
```

Comment:
`pkijs` and all other code related to the workaround should be removed once I've fixed the root issue.